ts/components/wallet/layout/Touchable.tsx

```diff
@@ -0,0 +1,23 @@
+/**
+ * Returns the touchable component wrapped by a Row
+ */
+import * as React from "react";
+import { Row } from "react-native-easy-grid";
+import { hasTouchable, TopContent } from "./types";
+
+type Props = Readonly<{
+  content: TopContent;
+}>;
+
+export const TOUCHABLE_SIZE = 2;
+
+export class Touchable extends React.Component<Props> {
+  public render(): React.ReactNode {
+    if (hasTouchable(this.props.content)) {
```

Comment:
this logic should be implemented in the caller `(content ? <Touchable /> : null)`